deploy/lib/deployContainers.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+module.exports = {
+  deployContainers() {
+    this.serverless.cli.log('Deploying Containers...');
+    return BbPromise.bind(this)
+      .then(this.deployEachContainer)
+      .then(() => this.serverless.cli.log('Waiting for container deployments, this may take multiple minutes...'))
+      .then(this.waitContainersAreDeployed)
+      .catch((err) => {
+        throw new Error(err.response.data.message)
+      });
+  },
+
+  deployEachContainer() {
+    const promises = this.containers.map(
+      cont => this.provider.apiManager.post(`containers/${cont.id}/deploy`, {})
+      .then(response => response.data)
+      .catch((err) => {
+        throw new Error(err.response.data.message)
+      }),
+    );
+
+    return Promise.all(promises);
+  },
+
+  waitContainersAreDeployed() {
+    return this.provider.apiManager.get(`namespaces/${this.namespace.id}/containers`)
```

Comment:
we're re-formatting the url many times, you should create utility functions in `shared` to format URLs, like `listContainerUrl(namespaceId)` for example, this way if we have to change the container URL, we do it in one place and not multiple ones

Reply:
I don't think it is useful and we don't do it for functions.